docs/testing-contracts.rst

```diff
@@ -5,6 +5,7 @@ Testing a Contract
 
 For testing Vyper contracts we recommend the use of `pytest <https://docs.pytest.org/en/latest/contents.html>`_ along with one of the following packages:
 
+    * `Ape <https://github.com/ApeWorX/ape>`_: A development and testing framework for smart contracts targeting the Ethereum Virtual Machine
```

Comment:
```suggestion
    * `Ape <https://github.com/ApeWorX/ape>`_: The smart contract development tool for Pythonistas, Data Scientists, and Security Professionals
```

Reply:
Also curious if it might be better to link to our docs directly instead of creating separate copy that has to be kept up to date

https://docs.apeworx.io/ape/stable/userguides/testing.html

Reply:
+1 ^